flipperdevices/flipperzero-firmware
# Various improvements: Toolbox, Updater and Unit Tests.

# What's new

- Toolbox: add seek to character stream method
- UpdateUtils: reverse manifest iterator
- UnitTests: more unit tests
- Updater: remove empty folder from Manifest file

# Verification 

- Run unit tests
- Enable debug logging 
- Update with `flash_usb_full` and carefully check serial logs especially part when we removing folders
- Review the code

# Checklist (For Reviewer)
